view/template/content/roadmap.php

```diff
@@ -6,6 +6,12 @@
   lbry.roadmap('#project-roadmap');
 <?php js_end() ?>
 <main>
+  <?php if (Config::get('github_key') == null || Config::get('is_prod') === "no"){
```

Comment:
1) Standards wise, this check should probably be in controller.
2) The `'is_prod'` check seems ugly / mistake prone. I checked `is_prod` setting on production web server and it is `"yes"` as a string, which _also_ seems ugly. @lyoshenka can you coordinate with @maximest-pierre on lbry.io access? This will let you see production config values. Likely `is_prod` should be boolean true, and checks for production should demand `=== true`.

Reply:
I am implementing the check inside of a controller right now. I am opening an issue for the is_prod issue.